Tasks/CondaEnvironmentV0/task.json

```diff
@@ -29,17 +29,18 @@
             "name": "environmentName",
             "type": "string",
             "label": "Environment name",
-            "required": true,
+            "required": false,
             "defaultValue": "",
-            "helpMarkDown": "Name of the Conda environment to create and activate."
+            "helpMarkDown": "Name of the Conda environment to create and activate. If left empty, subsequent build steps will run in the global `base` environment."
         },
         {
             "name": "packageSpecs",
             "type": "string",
             "label": "Package specs",
             "required": false,
             "defaultValue": "python=3",
-            "helpMarkDown": "Space-delimited list of packages to install when creating the environment."
+            "visibileIf": "environmentName != \"\"",
```

Comment:
Type, `visibile`

Reply:
Weird, I had actually deleted that line.  Let met try to fix it.